### PR TITLE
Update localize.ts

### PR DESCRIPTION
**Pull request by misa1515**

### CODE REVIEW

**Review by swingerman — CHANGES_REQUESTED** [review]
It seems that you forgot to add the translation file.

**Comment by misa1515** [comment]
Hello,
normally (like for other projects) are attached (please see screenshot)
image

File: src/localize/localize.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ import * as en from './languages/en.json';
 import * as nb from './languages/nb.json';
 import * as ptbr from './languages/ptbr.json';
 import * as ptpt from './languages/ptpt.json';
+import * as sk from './languages/sk.json';
```
Comment (swingerman) [inline src/localize/localize.ts:5]: This file doesn't exists

**Comment by misa1515** [comment]
understood, it is in 2nd pull request.
image
If you eplain me "how to", I can make it, in future at once.

**Comment by swingerman** [comment]
Please create one PR with all the files included.

**Review by swingerman — APPROVED** [review]

**Pull request merged**